tests/test_Dictionary.py

```diff
@@ -1,6 +1,8 @@
 """Test the Dictionary class methods."""
 import ee
 
+import geetools  # noqa
```

Comment:
You also need to specify which one you want to ignore: 
```suggestion
import geetools  # noqa: F401
```